shell/platform/fuchsia/flutter/surface.cc

```diff
@@ -12,8 +12,9 @@
 
 namespace flutter_runner {
 
-Surface::Surface(std::string debug_label)
-    : debug_label_(std::move(debug_label)) {}
+Surface::Surface(std::string debug_label,
+                 flutter::ExternalViewEmbedder* view_embedder)
```

Comment:
The raw pointer here is a little scary, but the next CL I'll upload will turn this into a shared_ptr so it should be OK

Reply:
Ack